docker-compose-staging.yml

```diff
@@ -94,3 +94,29 @@ services:
     build:
       context: ./
       dockerfile: docker/prod/code-upload-worker/Dockerfile
+
+  prometheus:
+    image: prom/prometheus:latest
+    user: "1000"
```

Comment:
@burnerlee What is the use of `user` here?

Reply:
@RishabhJain2018 its to provide right user id to grant privileges to make changes in file system, it was throwing error otherwise